gno.land/pkg/gnoclient/client_test.go

```diff
@@ -359,3 +360,188 @@ func TestClient_Call_Errors(t *testing.T) {
 		})
 	}
 }
+
+// Run tests
+func TestClient_Run_Errors(t *testing.T) {
```

Comment:
Do you want to add an integ test for `Run`, too?

Reply:
Added: [5f6f619](https://github.com/gnolang/gno/pull/1637/commits/5f6f61991db14a174d8e8fc8a6e288f8e75a94f5)